src/sidebar/components/Annotation/AnnotationQuote.js

```diff
@@ -61,15 +26,18 @@ function QuotedText({ children, classes, style }) {
 function AnnotationQuote({ quote, isFocused, isOrphan, settings = {} }) {
   return (
     <Excerpt collapsedHeight={35} inlineControls={true} overflowThreshold={20}>
-      <QuotedText
-        classes={classnames({
-          'border-l-blue-quote': isFocused,
-          'line-through grayscale contrast-50': isOrphan,
-        })}
-        style={applyTheme(['selectionFontFamily'], settings)}
+      <StyledText
+        classes={classnames({ 'line-through grayscale contrast-50': isOrphan })}
       >
-        {quote}
-      </QuotedText>
+        <blockquote
+          className={classnames('hover:border-l-blue-quote', {
+            'border-l-blue-quote': isFocused,
+          })}
+          style={applyTheme(['selectionFontFamily'], settings)}
+        >
+          {quote}
+        </blockquote>
+      </StyledText>
```

Comment:
I'll explain this change a little in case it's useful information. These changes now appropriately (IMO) separate the styling concerns of the pieces at play here:

* It's no longer necessary to apply so many `className`s to the `blockquote` element as it will correctly pick up common `blockquote` styling from `StyledText`. The classes applied pertain to the specific styling of this `blockquote` in an `AnnotationQuote` context (blue left-border on hover or focus).
* I moved `dir` and `lang` attributes into `StyledText`

After doing refactoring to use `StyledText`, it became clear the local `QuotedText` component was no longer useful.